package.json

```
@@ -25,6 +25,7 @@
     "test": "npm run test:jest",
     "test:jest": "jest --verbose --passWithNoTests --env=jsdom",
     "test:ct": "cypress run --component",
+    "test:cto": "cypress open",
```

Let's make it consistent with other apps that already introduced this script:
```suggestion
    "test:openct": "cypress open --component",
```